packages/docs/vite.config.mts

```diff
@@ -236,6 +236,27 @@ export default defineConfig(({ command, mode, isSsrBuild }) => {
         }
       },
 
+      {
+        name: 'vuetify:mdi-js-icons',
+        enforce: 'pre',
+        resolveId(id) {
+          if (id === 'virtual:mdi-js-icons') {
+            return `\0virtual:mdi-js-icons`
+          }
+        },
+        async load (id) {
+          if (id === `\0virtual:mdi-js-icons`) {
+            const icons: { name: string, aliases: string[] }[] = await import('@mdi/svg/meta.json', {
```

Comment:
Use read file instead dynamic import, depending on node version this can fail

Reply:
We already use this in api-gen and enforce node >18